client/src/components/QandA/QuestionsList/HelpfulTracker.jsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react';
+// import { token, url } from '/config.js';
+import axios from 'axios';
+const localHost = 'http://127.0.0.1:3000';
+
+// class HelpfulTracker extends React.Component {
+//   constructor(props) {
+//     super(props);
+//     this.state = {
+//       yesScore: this.props.helpful,
+//       yesClicked: false
+//     }
+//   }
+// }
+
+const HelpfulTracker = ({ question_id, helpful, usage, reported }) => {
+  const [yesCount, setYesCount] = useState(helpful);
+  const [yesClicked, setYesClicked] = useState(false);
+  const [reportState, setReportState] = useState(<u>Report</u>);
```

Comment:
this is a dope idea for the report feature!

Reply:
Oh yeah! Thanks!